cli/cmd/cluster/cluster.go

```diff
@@ -309,9 +310,9 @@ func (c controlplaneUpdater) ensureComponent(component, namespace string) error
 	}
 
 	histClient := action.NewHistory(actionConfig)
-	histClient.Max = 1
+	histMax := 1
```

Comment:
What if the last one in the history was not a successful deploy? Do we want the last one in the history or the last successful deploy in the history?

Reply:
Good point :neutral_face:

Reply:
We could check for the last "deployed" instance.

Reply:
I'd say it's out of scope for this PR? Maybe create an issue and create a follow-up PR? This PR fixes existing assumed functionality, while suggestion from @surajssd changes the behavior from existing one, although I agree what we do right now is not fully correct.

Reply:
Alright sure, lets move forward. But yeah we will need a func that give the last successful deploy.

Reply:
https://github.com/kinvolk/lokomotive/issues/1520